### PR TITLE
better pagination for larger number of blog posts

### PR DESCRIPTION
**Pull request by sunnysideup**
I have a client who has 2000 entries on their blog without PaginationSummary this looks madness.

### CODE REVIEW

File: templates/Includes/Pagination.ss [hunk 1]
```diff
@@ -6,7 +6,7 @@
 			<a class="prev" href="{$PrevLink}">&larr;</a>
 		<% end_if %>
 
-		<% loop $Pages %>
+		<% loop PaginationSummary(4) %>
```
Comment (micmania1) [inline templates/Includes/Pagination.ss]: Can you add the `$` at the start please (3.x syntax).

Comment (sunnysideup) [inline templates/Includes/Pagination.ss]: done.

On 15 December 2015 at 09:55, Michael Strong <notifications@github.com>
wrote:

> In templates/Includes/Pagination.ss
> <https://github.com/silverstripe/silverstripe-blog/pull/351#discussion_r47556202>
> :
>
> > @@ -6,7 +6,7 @@
> >  			<a class="prev" href="{$PrevLink}">&larr;</a>
> >  		<% end_if %>
> >
> > -		<% loop $Pages %>
> > +		<% loop PaginationSummary(4) %>
>
> Can you add the $ at the start please (3.x syntax).
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/silverstripe/silverstripe-blog/pull/351/files#r47556202>
> .
>



-- 
Nicolaas Thiemen Francken
  www.sunnysideup.co.nz
  phone: +64221697577

**Comment by tractorcow** [comment]
Any chance you could squash those two commits please?

**Comment by sunnysideup** [comment]
Hi

I made a new pull request.  I hope I did this right. I am just trying to
use the GIT interface as doing it this way would save time I feel.

On 15 December 2015 at 15:44, Damian Mooyman <notifications@github.com>
wrote:

> Any chance you could squash those two commits please?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/silverstripe/silverstripe-blog/pull/351#issuecomment-164626336>
> .
>



-- 
Nicolaas Thiemen Francken
  www.sunnysideup.co.nz
  phone: +64221697577

**Comment by tractorcow** [comment]
Merged with https://github.com/silverstripe/silverstripe-blog/pull/352

**Pull request closed without merging**